### PR TITLE
Feature/employee manager

### PR DESCRIPTION
**Pull request by EmilPPrpic**

### CODE REVIEW

File: config/config.yaml [hunk 1]
```diff
@@ -11,11 +11,11 @@ global:
 
     ENVIRONMENT_NAME: {{ env.ENVIRONMENT_NAME }}
     MODULE: '{{ env.MODULE | default("local", True) }}'
-    LOG_LEVEL: INFO
+    LOG_LEVEL: DEBUG
```
Comment (assert-0) [inline config/config.yaml]: Return to INFO.

File: config/config.yaml [hunk 2]
```diff
@@ -162,3 +163,22 @@ services:
     resources:
       cpu: 50m
       memory: 75Mi
+  employee-manager:
```
Comment (assert-0) [inline config/config.yaml:169]: Add newline.

File: config/local.config.yaml [hunk 3]
```diff
@@ -15,15 +15,19 @@ global:
...
     port: 8001
+
   armory:
     command:
       build: "./build.sh"
       run: "armory-admin runserver"
     env:
       SITE_URL: '{{ env.ARMORY_SITE_URL | default("http://localhost:8000", true) }}'
     port: 8000
+  employee-manager:
```
Comment (assert-0) [inline config/local.config.yaml:33]: Newline.

File: services/employee_manager/apps/employees/admin.py [hunk 4]
```diff
@@ -0,0 +1,16 @@
+from django.contrib import admin
+from .models import Employee
+from import_export.admin import ImportExportModelAdmin
+from django.contrib import admin
+
+
+
```
Comment (assert-0) [inline services/employee_manager/apps/employees/admin.py:5]: Remove 2 newlines.

File: services/employee_manager/apps/employees/admin.py [hunk 5]
```diff
@@ -0,0 +1,16 @@
+from django.contrib import admin
```
Comment (assert-0) [inline services/employee_manager/apps/employees/admin.py:1]: Reorder imports.

File: services/employee_manager/apps/employees/models.py [hunk 6]
```diff
@@ -0,0 +1,29 @@
+from django.db import models
```
Comment (assert-0) [inline services/employee_manager/apps/employees/models.py:1]: Reorder imports.

File: services/employee_manager/apps/service.py [hunk 7]
```diff
@@ -0,0 +1,10 @@
+import logging
```
Comment (assert-0) [inline services/employee_manager/apps/service.py:1]: Should we remove this file @mjuraj ?

File: services/employee_manager/apps/employees/models.py [hunk 8]
```diff
@@ -0,0 +1,30 @@
...
+from django.db.models.signals import post_save
+from django.dispatch import receiver
+
+from services.employee_manager.api.views import Employee as EmployeeView
+
+
+class Employee(models.Model):
+    id = models.AutoField(primary_key=True, db_index=True)
+    firstName = models.CharField(max_length=200)
+    lastName = models.CharField(max_length=200)
```
Comment (mjuraj) [inline services/employee_manager/apps/employees/models.py]: in snake_case

File: services/employee_manager/apps/employees/models.py [hunk 9]
```diff
@@ -0,0 +1,30 @@
...
+            "id": self.id,
+            "firstName": self.firstName,
+            "lastName": self.lastName,
+            "email": self.email,
+            "active": self.active
+        }
+
+
+@receiver(post_save, sender=Employee)
+def Employeeupdated(sender, instance: Employee, *args, **kwargs):
```
Comment (mjuraj) [inline services/employee_manager/apps/employees/models.py]: function names are in snake_case. Also, say what the function does, not what triggers it

**Review by mjuraj — APPROVED** [review]

**Pull request merged**